src/ray/raylet/worker_pool.cc

```diff
@@ -159,6 +153,19 @@ WorkerPool::~WorkerPool() {
   }
 }
 
+void WorkerPool::Start() {
+  if (RayConfig::instance().kill_idle_workers_interval_ms() > 0) {
+    periodical_runner_.RunFnPeriodically(
+        [this] { TryKillingIdleWorkers(); },
+        RayConfig::instance().kill_idle_workers_interval_ms(),
+        "RayletWorkerPool.deadline_timer.kill_idle_workers");
+  }
+
+  if (RayConfig::instance().enable_worker_prestart()) {
```

Comment:
is it too much ask to add another flag controls when prestart happens (raylet or first driver registers), and default to first driver registers?